setup_board

```diff
@@ -155,6 +155,12 @@ fi
 if [ "${FLAGS_skip_toolchain_update}" -eq "${FLAGS_TRUE}" ]; then
   UPDATE_ARGS+=" --skip_toolchain_update"
 fi
+if [[ "${BOARD}" == "arm64-usr" ]]; then
+  [[ ! -d /etc/portage/package.use ]] && sudo mkdir /etc/portage/package.use
+  sudo_clobber "/etc/portage/package.use/grub" <<EOF
+sys-boot/grub grub_platforms_arm64
+EOF
+fi
```

Comment:
I think it is too much trouble to run emerge by hand, so I added this back in with PR https://github.com/coreos/scripts/pull/478.